sourceDocumentName doesn't work on sub folders recursively

Hi,

is it possible to use a maven configuration like (it seems not):
````
<sourceDocumentName>**/a_main.adoc</sourceDocumentName>
````
The idea is to have a project structure like:
src/main/asciidoc/subfolder1/a_main.adoc
src/main/asciidoc/subfolder2/a_main.adoc
etc.

All sub folders contain more *.adoc which are included in the a_main.adoc.

Would be nice if that works.

Right now I use
````
<execution>
	<id>output-pdf</id>
	<phase>process-resources</phase>
	<goals>
		<goal>process-asciidoc</goal>
	</goals>
	<configuration>
		<backend>pdf</backend>
		<sourceHighlighter>coderay</sourceHighlighter>
		<preserveDirectories>true</preserveDirectories>
		<sourceDocumentName>**/a_main.adoc</sourceDocumentName>
		<relativeBaseDir>true</relativeBaseDir>
		<outputDirectory>${basedir}/target/pdf</outputDirectory>
		<imagesDir>${basedir}/target/pdf/images</imagesDir>
		<attributes>
			<icons>font</icons>
			<pagenums/>
			<toc/>
			<idprefix/>
			<idseparator>-</idseparator>
			<docinfo1>true</docinfo1>
		</attributes>
	</configuration>
</execution>
````

But this throws an error:
````
ERROR] Failed to execute goal org.asciidoctor:asciidoctor-maven-plugin:1.5.6:process-asciidoc (output-pdf) on project architecture-docu: Unable to locate output directory: Invalid argument -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
````

Regards
Steffen

--- a/README.adoc
+++ b/README.adoc
@@ -107,6 +107,14 @@ As this is a typical Maven plugin, simply declare the plugin in the `<plugins>` 
 There are several configuration options that the Asciidoctor Maven plugin accepts, which parallel the options in Asciidoctor:
 
 sourceDirectory:: defaults to [.path]_$\{basedir}/src/main/asciidoc_
+
+[NOTE]
+====
+All paths and AsciiDoc documents that start with `pass:[_]` are considered _internal_ and are skipped.
+That is, AsciiDocs are not rendered and resources are not copied to target, but you can include them normally from other AsciiDocs. +
+This is useful to split your sources in smaller files of master documents and included parts.
+====
+
 sourceDocumentName:: an override to process a single source file; defaults to all files in `$\{sourceDirectory}`
 sourceDocumentExtensions:: (named `extensions` in v1.5.3 and below) a `List<String>` of non-standard file extensions to render.
 Currently ad, adoc, and asciidoc will be rendered by default

--- a/README.adoc
+++ b/README.adoc
@@ -112,7 +112,7 @@ sourceDirectory:: defaults to [.path]_$\{basedir}/src/main/asciidoc_
 ====
 All paths and AsciiDoc documents that start with `pass:[_]` are considered _internal_ and are skipped.
 That is, AsciiDocs are not rendered and resources are not copied to target, but you can include them normally from other AsciiDocs. +
-This is useful to split your sources in smaller files of master documents and included parts.
+This is useful to split your sources in sets of master documents and included parts.
 ====
 
 sourceDocumentName:: an override to process a single source file; defaults to all files in `$\{sourceDirectory}`

